README.md

```diff
@@ -24,6 +24,10 @@ $ make prepare.ir
 Enter account NTrezR3C4X8aMLVg7vozt5wguyNfFhwuFx password >
 Sent invocation transaction fa6ba62bffb04030d303dcc95bda7413e03aa3c7e6ca9c2f999d65db9ec9b82c
 ```
+Also you can add self-signed node (`s04.neofs.devenv`) certificate to truststore:
```

Comment:
As far as I see, you **should** add self-signed certificate, otherwise clients won't be able to connect to s04, will they? 

Probably we can set `NEOFS_GRPC_TLS_ENABLED=true` in s04 config at `make prepare.storage`. Then it will be optional.